pkg/tfbridge/info.go

```diff
@@ -716,3 +718,16 @@ func (m *MarshallableProviderInfo) Unmarshal() *ProviderInfo {
 
 	return &info
 }
+
+// Calculates the major version of a go sdk
```

Comment:
Maybe add a comment that it ignores v1 and why

Reply:
Good call! Updated now